app/helpers/application_helper.rb

```diff
@@ -10,4 +12,50 @@ def user_status(flag)
   def url_for_client
     current_user.admin? ? admin_clients_url : clients_url
   end
+
+  def url_for_project
+    current_user.admin? ? admin_projects_url : projects_url
+  end
+
+  def url_for_root
+    current_user.admin? ? admin_projects_url : root_url
+  end
+
+  def authenticate_manager
+    current_user&.manager?
+  end
+
+  def flash_class(level)
+    case level
+    when 'notice' then 'alert alert-info'
+    when 'alert' then 'alert alert-danger'
+    end
+  end
+
+  def error_class(resource, field)
+    resource.errors[field].any? ? 'form-group has-danger' : 'form-group'
```

Comment:
form-group should not be part of this check.
could be something like.....
 'has-danger' if resource.errors[field].any?